cmd/kubehound/ingest.go

```diff
@@ -59,6 +59,27 @@ var (
 			return core.CoreClientGRPCIngest(khCfg.Ingestor, khCfg.Ingestor.ClusterName, khCfg.Ingestor.RunID)
 		},
 	}
+
+	remoteIngestRehydrateCmd = &cobra.Command{
```

Comment:
UX question:
Woudn't it be better to have a flag like `--latest` or even just `latest` on the `ingest` subcommand?

rehydrate is "another command to know" instead of "wait, I just want to ingest whatever the latest thing is but don't remember the exact path, can I just use latest and be done with it?"

Reply:
yes but the ingest require the --run_id and --cluster which is not needed, so this is why I split it.

Reply:
I see, then a couple possible solution that, imo would be better, if you don't want to spend time on it, i'm fine, but that'll be harder to change later I guess :D

I think it'd be better to have:
```bash
kh ingest remote latest # (or even, maybe without latest since that could be default value?) This also allows for autocomplete
kh ingest remote run_xyz --cluster xyz
kh ingest remote latest --cluster xyz

# also allow to ingest, depending on the env, cluster that matters to the user, individually
kh ingest remote latest --cluster xyz --cluster abc 
(or, possibly)
kh ingest remote latest --cluster xyz,abc 
```

Or, even to keep it more aligned with the current run_id flag system, just checking if `run_id==latest` would work
(I think it would prefer the non flag version, because the auto completion would be simpler / expected, but that's a bit more work, so i'm find with the proposal below for now)
```bash
kh ingest remote --run_id=latest
kh ingest remote --run_id=run_xyz --cluster xyz
kh ingest remote --run_id=latest --cluster xyz
```


The current PR adds, imo, confusion about what to run and makes the feature less discover-able
```bash
kh ingest remote rehydrate # (how do you say you want to get specific cluster? e.g: If you have 500 cluster dumped, and your team manages only 4, I don't think you'll want to ingest them all?)
kh ingest remote --run_id=run_xyz --cluster xyz
```

Rationale for the move from flags to positional arg: `--cluster` *is now not* mandatory (as in: it can be found by itself during "rehydrate") but `--run_id` *is* mandatory? (as in: it's either you provide it or set for you but there's one), so it feels weird to have the two